doc/source/conf.py

```diff
@@ -0,0 +1,56 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('development/'))
+sys.setrecursionlimit(1500)
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'pandas-mentoring'
+copyright = '2019, datapythonista'
+author = 'datapythonista'
```

Comment:
I'm surely not the author. ;) We need a team name, in the meanwhile may be you can simply say `pandas mentoring team` or something?

Reply:
I'll change this to `pandas mentoring team`